app/gui/src/dashboard/providers/__test__/DriveProvider.test.tsx

```diff
@@ -1,83 +0,0 @@
-import { DirectoryId } from '#/services/Backend'
```

Comment:
Why did we remove the tests?

Reply:
behavior has been changed so these tests are no longer relevant

we can write a new test in the same style if that works?

Reply:
Yes for sure. Unrelevant tests can be removed, but we need to add new ones if we add a `new` behavior. Coverage shouldn't decrease